test/createAzureResource.test.ts

```diff
@@ -0,0 +1,103 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as assert from 'assert';
+import { ResourceManagementClient } from 'azure-arm-resource';
+import { WebSiteManagementClient, WebSiteManagementModels } from 'azure-arm-website';
+import { IHookCallbackContext, ISuiteCallbackContext } from 'mocha';
+import * as vscode from 'vscode';
+import { deleteDefaultResourceGroup, longRunningTestsEnabled } from './global.test';
+import { AzureTreeDataProvider, constants, DialogResponses, ext, getRandomHexString, ILinuxRuntimeStack, SiteRuntimeStep, TestAzureAccount, TestUserInput, WebAppProvider } from '../extension.bundle';
+
+suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Promise<void> {
+    this.timeout(1200 * 1000);
+    const resourceGroupsToDelete: string[] = [];
+    const testAccount: TestAzureAccount = new TestAzureAccount();
+    // tslint:disable-next-line:no-backbone-get-set-outside-model
+    const oldAdvancedCreationSetting: boolean | undefined = <boolean>vscode.workspace.getConfiguration(constants.extensionPrefix).get('advancedCreation');
+    const runtimeLTS: ILinuxRuntimeStack | undefined = new SiteRuntimeStep().getLinuxRuntimeStack().find((runtime: ILinuxRuntimeStack) => {
+        return runtime.displayName.includes('LTS - Recommended for new apps');
```

Comment:
I suppose this fixes this specific case, but I would prefer to see a more generic solution that helps other test authors as well.

For example, change the constructor of `TestUserInput` from `constructor(inputs: (string | undefined)[]) {` to `constructor(inputs: (string | RegExp | undefined)[]) {` so that you can just pass in a RegExp with 'LTS' in it